### ISSUE TITLE
Can not run ./SeamCut 5 1 400 1 0.2

### ISSUE THREAD

**Issue by lukesonglu** [SuTanTank/VideoStitchingViaShakinessRemoving#4]

**Comment by SuTanTank** [comment]
Did you compile the seamcut project? any error message?

**Comment by lukesonglu** [comment]
Yes, I copy the seamcut.exe to the folder of output frame but when I try to run it, it just shut down immediately.

**Comment by lukesonglu** [comment]
How can I get the result "D1.png D2.png....."

**Comment by SuTanTank** [comment]
Well since you can compile it, why not try run it in Visual Studio or any other debugger... It may be a bug, or wrong filename...
